docs/getting-started.md

```diff
@@ -97,13 +97,29 @@ integration libraries.
 
 ## Setting up backend and running
 
-Newer versions of Web (> 7.0.2) can only run against [oCIS](https://github.com/owncloud/ocis), whereas older versions (< 7.1.0) can run against [ownCloud 10](https://github.com/owncloud/core/) as well.
-Depending which one you chose, please check the matching section:
+Newer versions of Web (> 7.0.2) can only run against [oCIS](https://github.com/owncloud/ocis).
 
-- [Setting up with oCIS as backend]({{< ref "backend-ocis.md" >}})
-- [Setting up with ownCloud as backend]({{< ref "backend-oc10.md" >}})
+### Setting up Web
 
-## Running
+- Clone the [repository](https://github.com/owncloud/web/)
+- Initally install all dependencies by running `pnpm install`
+- Copy `./config/config.json.sample-ocis` to `./config/config.json` and adjust values if required
 
-- [Running with oCIS as backend]({{< ref "backend-ocis.md#running-web" >}})
-- [Running with ownCloud as backend]({{< ref "backend-oc10.md#running-web" >}})
+### Running Web
+
+- Start bundling web with a watcher by running `pnpm build:w`
+
+### Setting up oCIS
+
+- Setup oCIS by following the [setup instructions](https://owncloud.dev/ocis/getting-started/)
+- Start oCIS with local links to your bundled web frontend and config by running `WEB_ASSET_CORE_PATH=../../web/dist WEB_UI_CONFIG_FILE=../../web/dist/config.json OCIS_INSECURE=true IDM_CREATE_DEMO_USERS=true ./bin/ocis server` (and make sure to adjust paths as necessary)
```

Comment:
This guide is not about setting up a production ocis with web, but about setting up a working dev stack. Shouldn't we just explain the docker-compose stack from the monorepo then? I'd like to refrain from describing an ocis binary with web setup (that can also be needed, but it's more of an edge case...)